app.py

```diff
@@ -127,53 +133,82 @@ def message(event, client):
                 # compares the thread message history to the individual questions. 
                 # if a the exact string of a question is in message["text"], change the value of last_question_asked accordingly.
                 for message in thread_message_history:
-                    if message["text"] == bot_messages.question_one:
+                    if message["text"] == bot_messages.questions[0]:
                         last_question_asked = 1
-                    elif message["text"] == bot_messages.question_two:
+                    elif message["text"] == bot_messages.questions[1]:
                         last_question_asked = 2
-                    elif message["text"] == bot_messages.question_three:
+                    elif message["text"] == bot_messages.questions[2]:
                         last_question_asked = 3
 
-                # bot records response to question 1 and asks question 2 afterwards.
+
+                # bot records response (the wiki title) and asks question 2 afterwards.
                 if last_question_asked == 1:
-                    client.chat_postMessage(channel=channel_id, text=bot_messages.response_recorded, thread_ts=message_thread_ts)
-                    client.chat_postMessage(channel=channel_id, text=bot_messages.question_two, thread_ts=message_ts)
-                    file_write(f"# {text}\n")
-                # bot records response to question 2 and asks question 3 afterwards.
+                    if not title_in_documentation(text):
+                        client.chat_postMessage(channel=channel_id, text=bot_messages.response_recorded, thread_ts=message_thread_ts)
+                        client.chat_postMessage(channel=channel_id, text=bot_messages.questions[1], thread_ts=message_ts)
+                        wiki_title = text
+                    else:
+                        client.chat_postMessage(channel=channel_id, text=bot_messages.already_titled, thread_ts=message_ts)
+                # bot writes response to question 2 into markdown file and asks question 3 afterwards.

```

Comment:
Nit: would be helpful to add a brief comment about what each question is asking for right before you post it (i.e. question one asking for title, 2 for picture, 3 for supplemental information)

Reply:
I was debating between writing either just "question 3" or what the question asked for, but now that you point it out, the latter definitely makes more sense.